docs/UserGuide.md

```diff
@@ -130,6 +133,16 @@ The app helps students to consolidate important data, such as profiles of profes
 
 <div style="page-break-after: always;"></div>
 
+## UI Mockup
```

Comment:
sorry i was not clear about it but can change point 4 at line 128 from `resembling the UI shown in the Home page` to `resembling the UI shown in the [UI mockup](#ui-mockup) below`